web-terminal/values.yaml

```diff
@@ -0,0 +1,146 @@
+# Default values for web-terminal.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+bootstrapCommands: |
+  apk add --quiet --no-progress curl ttyd aws-cli kubectl bash tar
+  STERN_VERSION=$(curl -s https://api.github.com/repos/stern/stern/releases/latest | grep 'tag_name' | cut -d '"' -f 4)
+  curl -LO https://github.com/stern/stern/releases/download/${STERN_VERSION}/stern_${STERN_VERSION#v}_linux_amd64.tar.gz
+  tar -xvzf stern_${STERN_VERSION#v}_linux_amd64.tar.gz && mv stern /usr/local/bin/ && chmod +x /usr/local/bin/stern
+
+command: bash -c "echo hello world"
+# command: stern --since 24h "pod"
```

Comment:
**Question**: Should it be the default command?

Reply:
I am trying to make the web-terminal chart agnostic.

Also, when using the chart in an helmfile, we are supposed to override the command to specify which pods we want to watch (see the demo helmfile).